### ISSUE TITLE
Images not showing 

### ISSUE THREAD

**Issue by Alnaggar94** [troychaplin79/wp-multitenant#2]
Hello, I have been testing and everything works great except the images not showing in media library or in the frontend, I have been search for a slotion and couldn't figure it out.

The problem seems related to .htaccess rules or maybe permission not set right, Because when I check the upload folder through FTP it's empty? and the images show when I edited them in the media library! and when I go to the image URL it redirects me to the home page.

See this video record - https://imgur.com/LzkGJlm

I have set the permission as the following:

For domain folder:
```
sudo chmod -R 755 /var/www/natureresult.net
sudo chown -R www-data:www-data /var/www/natureresult.net
```
For wp-multitenant folder:
```
sudo chmod -R 755 /var/www/wp-multitenant
sudo chown -R www-data:www-data /var/www/wp-multitenant
```

**Comment by Alnaggar94** [comment]
@troychaplin79 - I just found out that images get uploaded to `/var/www/wp-multitenant/assets/uploads` and it supoosed to be in `/var/www/natureresult.net/wp-content/uploads/` . Please check from your end.

![pppp](https://user-images.githubusercontent.com/49800146/128598961-c065c09d-5529-4cc8-8f8a-4ad56a19b87e.JPG)

**Comment by Alnaggar94** [comment]
I found the solution, You forget to define the upload path folder in `wp-constants.php ` I have added `define('UPLOADS', ''.'wp-content/uploads');` to `wp-constants.php` and everything is work great. Again thank you so much for this great project.

**Comment by Alnaggar94** [comment]
@troychaplin79 - Actually this fixed the problem partly, the images now showing right in the backend and site but they get uploaded to `wp-multitenant/app/stable/wp-content/uploads`, and they are supposed to be in `/var/www/natureresult.net/wp-content/uploads/` any solution to correct this behavior?

**Comment by troychaplin** [comment]
Fixed, tested and updated to version 1.1.8 in master
